src/app/build.gradle

```diff
@@ -16,12 +16,6 @@ jacocoTestReport {
 		xml.enabled = true
 		csv.enabled = false
 	}
-
-	afterEvaluate {
-		classDirectories = files(classDirectories.files.collect {
```

Comment:
I had previously excluded the APIClient from tests for reasons best known to past me. Have now added a test class for it.